### ISSUE TITLE
Incorrect link to Facebook site

### ISSUE THREAD

**Issue by AndreySurzhan** [reactjs/react-tutorial#162]
> This is the React comment box example from[ the React tutorial](http://facebook.github.io/react/docs/tutorial.html).

The link does lead to comment box tutorial on Facebook site

**Comment by vikneshwar** [comment]
Yeah , i was about to raise the issue , i was half way through the comment box tutorial , now i cant find it anywhere

**Comment by AndreySurzhan** [comment]
Here is the link with original tutorial 
http://chenglou.github.io/react/docs/tutorial.html

**Comment by wordyallen** [comment]
@AndreySurzhan it redirects to the new one.

**Comment by nishina555** [comment]
@AndreySurzhan 
[this link](https://github.com/facebook/react/blob/v15.3.2/docs/docs/tutorial.md) contains comment box tutorial so I put in the pull request here: #166
